projects/storefrontapp-e2e-cypress/cypress/e2e/regression/my-account/my-account-v2-orders.e2e.cy.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { fillLoginForm } from '../../../helpers/auth-forms';
+import { viewportContext } from '../../../helpers/viewport-context';
+import { isolateTests } from '../../../support/utils/test-isolation';
+
+describe(
+  'My Account Version-2 Order History and Order Details',
+  { testIsolation: false },
+  () => {
+    viewportContext(['desktop'], () => {
+      isolateTests();
+      before(() => {
+        cy.window().then((win) => win.sessionStorage.clear());
+        cy.visit('/');
+      });
+
+      beforeEach(() => {
+        cy.restoreLocalStorage();
+      });
+
+      it('should navigate to login page and SignIn with user details', () => {
+        cy.findByText(/Sign in \/ Register/i).click();
+        fillLoginForm({ username: 'cdp.user@sap.com', password: 'Test@1' });
+      });
+
+      it('should navigate to Order History page', () => {
+        let totalCount;
+        cy.findByText(/My Account/i).click();
+        cy.findByText(/Order History/i).click();
+        cy.get('h2').contains('All Orders');
+        cy.get('.cx-myaccount-v2-order-history-body > .cx-each-order').then(
+          (value) => {
+            totalCount = Cypress.$(value).length;
+            expect(value).to.have.length(totalCount);
+          }
+        );
+      });
+
+      it('should navigate to view details of first order', () => {
+        cy.get('.cx-myaccount-v2-order-history-code > a').first().click();
```

Comment:
[MEDIUM]
Is this test still passing for you locally after the renaming `myaccount` -> `my-account`? If yes, then it would be a smell.

I guess we need to fix the css selector in the test: `cx-myaccount-v2-order-history-body` -> `cx-my-account-v2-order-history-body`

Reply:
Let me check, because when I wrote this test case it was 'cx-myaccount-v2-order-history-body'.
will check & update accordingly

Reply:
done